packages/components/src/components/tooltip/tooltip.js

```diff
@@ -78,6 +78,12 @@ const getMenuOffset = (menuBody, menuDirection) => {
   return undefined;
 };
 
+// @questions
+// -- What is the expected behavior if the user clicks on the trigger when the tooltip is already open
+// -- What's the focus state look like when there is only rich text within a tooltip (no buttons/links/etc)
```

Comment:
IIUC we don't anticipate that scenario from our designer's perspective. From technical perspective browser's default focus outline is good-enough escape hatch.

Reply:
@asudoh This is definitely a use-case for tooltips where they have rich text but do not contain buttons in them.  Because of the caret the default focus is actually kind of odd looking which is why I provided the link to how we handled this in Watson Health. I can definitely port those style updates over to Carbon if @carbon-design-system/design approves.

Reply:
Cool let's see what our designers think.